css/style.css

```diff
@@ -9,400 +14,622 @@ body {
-       left: 60%;
+    top: 30%;
+    width: 20%;
+    font-size: 50px;
+    line-height: 52px;
+    left: 55%;
+    }
+} 
+ 
+@media (max-width: 1500px) { 
+   .text {
+    width: 35%;
+    left: 55%;
+    height: 60%;
+     }
+}
+ 
+@media (max-width: 1500px) { 
+   .new-сollection__main-card {
+    width: 38%;
+    left: 55%;
+    margin-top: 1%;
+     }
+}
+     
+@media (max-width: 1500px) { 
+   .girl {
+    width: 50%;
+    right: 60%;
+     }
+}
+
+@media (max-width: 1500px) {
+   .new-сollection__social {
+    width: 45.5%;
    }
-    } 
+}
+
+@media (max-width: 1300px) {
+   .new-сollection__social {
+    width: 47%;
+   }
+}
+
+@media (min-width: 1270px) { 
+   .body {
+    margin: 0 auto;
+   }
+}
 
-@media (max-width: 1270px) { 
-    .text {
-        width: 35%;
-        left: 60%;
-        height: 60%;
+@media (max-width: 1200px) { 
+   .new-сollection__social {
+    width: 45%;
     }
 }
 
-@media (max-width: 1270px) { 
-    .new-сollection__main-card {
-        width: 38%;
-        left: 60%;
-        margin-top: 5%;
+@media (max-width: 1200px) { 
+   .girl {
+    width: 45%;
+    right: 60%;
     }
 }
-    
-@media (max-width: 1270px) { 
-    .girl {
-        width: 50%;
-        margin-left: 5%;
+ 
+@media (max-width: 1200px) { 
+   .new-сollection__main-button {
+    width: 35%;
+    font-size: 8px;
+     }
+}
+ 
+@media (max-width: 1200px) { 
+   .new-сollection__main-link {
+    width: 50%;
+    margin-left: 3%;
+    font-size: 10px;
+    margin-right: -15%;
+     }
+}
+ 
+@media (max-width: 1200px) { 
+   .new-сollection-arrow {
+    width: 10%;
+     } 
+}
+
+@media (max-width: 1200px) { 
+   .new-сollection__social {
+    width: 47%;
+     }
+}
+
+@media (max-width: 961px) { 
+   .new-сollection__main-card {
+    margin-top: 5%;
     }
 }
 
-@media (max-width: 1100px) { 
-    .new-сollection__main-button {
-        width: 35%;
+@media (max-width: 961px) { 
+   .new-сollection__main-link {
+    margin-right: -5%;
     }
 }
 
-@media (max-width: 1100px) { 
-    .new-сollection__main-link {
-        width: 50%;
-        margin-left: 3%;
+@media (max-width: 961px) { 
+   .new-collection__background {
+    max-width: 45%;
+    min-height: 900px;
     }
 }
 
-@media (max-width: 1100px) { 
-     .new-сollection-arrow {
-        width: 10%;
-    } 
-     }
+@media (max-width: 961px) { 
+   .logo {
+    margin-right: 20%;
+    }
+}
 
+@media (max-width: 961px) { 
+   .new-сollection-main-link {
+    margin-left: 5%;
+   } 
+}
 
 @media (min-width: 800px) { 
-    .logo-mobil,
-    .galka,
-    .new-collection__background_girl {
-        display: none;
-        } 
+   .logo-mobil,
+   .galka,
+   .new-collection__background_girl {
+    display: none;
     } 
-
+} 
+ 
 @media (max-width: 799px) { 
-    .logo,
-    .lang,
-    .check,
-    .new-сollection__social,
-    .text,
-    .new-сollection__main-card,
-    .new-collection__background {
-         display: none;
-        } 
+   .logo,
+   .lang,
+   .check,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)